plugins/shared-react/src/utils/user-timezone/user-timezone.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Gets the user time zone
+ *
+ * @return The users time zone as a string or 'UTC' by default
+ */
+export function getUserTimeZone(): string {
```

Comment:
Is this used anywhere? Seems like dead code to me.

Reply:
@tumido Resolved in add46e92718a8dc435d3eeae04a282fd26a90ffa